storage/innobase/handler/handler0alter.cc

```diff
@@ -92,6 +92,8 @@ static const alter_table_operations INNOBASE_ALTER_REBUILD
 	| ALTER_STORED_COLUMN_TYPE
 	*/
 	| INNOBASE_ALTER_VERSIONED_REBUILD
+	| ALTER_COLUMN_UNVERSIONED
+	| ALTER_COLUMN_EQUAL_PACK_LENGTH
 	;
```

Comment:
These flags do not really require the table to be rebuilt. I would not move them to `INNOBASE_ALTER_REBUILD`.

Reply:
That's the most easy solution. That's why I choose it. No so obvious - yes. But current state where some `REBUILD` are actually `REBUILD_BUT_SOMETIMES_INSTANT` is not obvious too.